test/end-to-end/test_supervisor_load_does_not_default_to_hab_user.ps1

```diff
@@ -0,0 +1,37 @@
+$username = "hab"
+$password = "Pass@word1"
+net user /add $username $password
+net localgroup administrators $username /add
+Add-Type -TypeDefinition (Get-Content ".expeditor\scripts\end_to_end\LsaWrapper.cs" | Out-String)
+$lsa_wrapper = New-Object -type LsaWrapper
+$lsa_wrapper.SetRight($username, "SeServiceLogonRight")
+
+$env:HAB_ORIGIN = "ci"
+hab origin key generate ci
+hab pkg build test/fixtures/windows_plans/dummy
+. .\results\last_build.ps1
+hab pkg install .\results\$pkg_artifact
+
+hab pkg install core/windows-service
+hab pkg exec core/windows-service install
+Start-Service Habitat
+Wait-Supervisor -Timeout 45
+
+Describe "hab svc load" {
+    $loadOut = hab svc load ci/dummy
+    Start-Sleep -Seconds 5
+
+    It "does not create a SVC_USRT metafile" {
+        Test-Path c:\hab\pkgs\$pkg_ident\SVC_USER | Should -Be $false
+    }
+    It "Succesfully loads service" {
+        $loadOut | Should -Be "The ci/dummy service was successfully loaded"
+    }
+    It "Reports service on HTTP Gateway as UP" {
+        ((Invoke-WebRequest "http://localhost:9631/services/dummy/default" -UseBasicParsing).content | ConvertFrom-Json).process.state | Should -Be "up"
+    }
+    It "runs hook as current user" {
+        $proc = Get-Process ping -IncludeUserName
```

Comment:
A note that the run hook is just continually running `ping` would make this more clear. It took me a minute to sort out what was going on.